content/blog/digib0y/2018-02-02_gci_2017_wrap_up_report.md

```diff
@@ -0,0 +1,56 @@
++++
+type = "blog"
+title = "GCI 2017 Wrap Up Report"
+author = "digib0y"
+date = "2018-02-02 00:13:41+05:30"
+tags = ["haiku", "software","GCI","Google","students","Code-In","Google Code-In","GCI2016"]
++++
+
+
+<p>Like all good things Google Code-In 2017 comes to an end, this year the program attracted over 3529 students in total for Google Code-In and out of those about ⅓ of the students claimed at least one task for Haiku, with 255 students completing at least one Haiku task. Haiku is also proud of its history of getting selected each year in <a href="http://codein.withgoogle.com">Google’s Code-In</a>.
```

Comment:
Officially I think it's [3555](https://opensource.googleblog.com/2018/01/google-code-in-2017-winners.html).

Reply:
Okay, will make required changes.